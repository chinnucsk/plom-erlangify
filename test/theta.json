{"value":{"E":{"partition_id":"variable_population","transformation":"logit","min":{"city1__all":0.07,"city2__all":0.07},"guess":{"city1__all":0.07,"city2__all":0.07},"max":{"city1__all":0.07,"city2__all":0.07},"sd_transf":{"city1__all":0,"city2__all":0}},"I":{"partition_id":"identical_population","transformation":"logit","min":0.000001,"guess":0.00001,"max":0.0001,"sd_transf":0.01},"v":{"partition_id":"identical_population","transformation":"log","unit":"D","type":"rate_as_duration","min":5,"guess":11,"max":20,"sd_transf":0.02}}}