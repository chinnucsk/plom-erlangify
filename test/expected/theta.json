{
  "parameter": {
    "v": {
      "partition_id": "identical_population",
      "transformation": "log",
      "unit": "D",
      "type": "rate_as_duration",
      "min": 5,
      "guess": 11,
      "max": 20,
      "sd_transf": 0.02
    },
    "E_0": {
      "partition_id": "variable_population",
      "group": {
        "city1__all": {
          "transformation": "logit",
          "min": 0.023333333333333334,
          "max": 0.023333333333333334,
          "guess": 0.023333333333333334,
          "sd_transf": 0
        },
        "city2__all": {
          "transformation": "logit",
          "min": 0.023333333333333334,
          "max": 0.023333333333333334,
          "guess": 0.023333333333333334,
          "sd_transf": 0
        }
      }
    },
    "E_1": {
      "partition_id": "variable_population",
      "group": {
        "city1__all": {
          "transformation": "logit",
          "min": 0.023333333333333334,
          "max": 0.023333333333333334,
          "guess": 0.023333333333333334,
          "sd_transf": 0
        },
        "city2__all": {
          "transformation": "logit",
          "min": 0.023333333333333334,
          "max": 0.023333333333333334,
          "guess": 0.023333333333333334,
          "sd_transf": 0
        }
      },
      "follow": "E_0"
    },
    "E_2": {
      "partition_id": "variable_population",
      "group": {
        "city1__all": {
          "transformation": "logit",
          "min": 0.023333333333333334,
          "max": 0.023333333333333334,
          "guess": 0.023333333333333334,
          "sd_transf": 0
        },
        "city2__all": {
          "transformation": "logit",
          "min": 0.023333333333333334,
          "max": 0.023333333333333334,
          "guess": 0.023333333333333334,
          "sd_transf": 0
        }
      },
      "follow": "E_0"
    },
    "I_0": {
      "partition_id": "identical_population",
      "transformation": "logit",
      "min": 5e-7,
      "guess": 0.000005,
      "max": 0.00005,
      "sd_transf": 0.005
    },
    "I_1": {
      "partition_id": "identical_population",
      "transformation": "logit",
      "min": 5e-7,
      "guess": 0.000005,
      "max": 0.00005,
      "sd_transf": 0.005,
      "follow": "I_0"
    }
  }
}